addons/account/models/account_journal_dashboard.py

```diff
@@ -82,7 +82,8 @@ def _query_has_sequence_holes(self):
              WHERE move.journal_id = ANY(%(journal_ids)s)
                AND move.state = 'posted'
           GROUP BY move.journal_id, move.sequence_prefix
-            HAVING COUNT(*) != MAX(move.sequence_number) - MIN(move.sequence_number) + 1
+            HAVING MIN(move.sequence_number) != 1
+                OR COUNT(*) != MAX(move.sequence_number) - MIN(move.sequence_number) + 1
```

Comment:
So, if you have
* 1
* 2
* 4

The min is 1, but there is a gap in 3; and that is not detected? :doubt:

Reply:
That case is indeed detected however this one is not:

- 3
- 4
- 5

(1 and 2 are missing)
The sequence should be starting at 1 hence the new condition

Reply:
It doesn't have to be. It is completely legal and some companies are doing it on purpose.

Reply:
@chklop 😇
I think it is still nice to have a warning shown when we are not starting at 1, it's a bit odd

Reply:
Moreover, in the list view, it is displayed in red as a warning so it helps with consistency with the dashboard
![image](https://user-images.githubusercontent.com/91618364/180420404-c11f34f6-afdc-4226-a99c-440004a60119.png)

Reply:
I think it's a nice tradeoff to have it in the list view but not on the dashboard.
You see that there is something wrong, but you don't have the big warning sign on the dashboard

But @chklop the floor is yours (you might have to fight with AVW and TSB depending on your answer)

Reply:
Did a double check and @william-andre is right, the requirement is to have a proper sequence, it doesn't have to necessarily start with a 1 (in fact, it can be anything, as long as it's a sequence). So I think we can leave like it was @rigr-odoo, and leave the "compromise" William is talking about.